test/browser/support/TestHarness.js

```diff
@@ -134,7 +134,7 @@ module.exports = class TestHarness {
           throw new Error(`Could not find schema ${schemaName}`);
         }
       } catch (error) {
-        console.error("Error validating event", error);
+        console.error('Error validating ${eventName} event', error);
```

Comment:
Are these backticks or single quotes? They look like single quotes on github but they'll need to be backticks

Reply:
Changes pushed